src/services/security/modules/RecipientAddressModule/index.test.ts

```diff
@@ -0,0 +1,1113 @@
+            },
+            type: 'UNKNOWN_ADDRESS',
+          },
+        ],
+      })
+    })
+
+    // multiSend
+    it('should warn about recipient(s) of multiSend recipients', async () => {
+      const multiSend = hexZeroPad('0x1', 20)
+
+      const recipient1 = hexZeroPad('0x2', 20)
+      const recipient2 = hexZeroPad('0x3', 20)
+
+      const data = getMockMultiSendCalldata([recipient1, recipient2])
+
+      const safeTransaction = createSafeTransaction({
+        to: multiSend,
+        data,
+        operation: OperationType.DelegateCall,
+      })
+
+      const result = await RecipientAddressModuleInstance.scanTransaction({
+        safeTransaction,
+        provider: mockProvider,
+        chainId: '1',
+        knownAddresses: [],
+      })
+
+      expect(isSmartContractSpy).toHaveBeenCalledTimes(2)
+      expect(mockGetBalance).toHaveBeenCalledTimes(2)
+      // Don't check as on mainnet
+      expect(mockGetSafeInfo).not.toHaveBeenCalled()
+
+      expect(result).toEqual({
+        severity: 1,
+        payload: [
+          {
+            severity: 1,
+            address: recipient1,
+            description: {
+              short: 'Address is not known',
+              long: 'The address is not an owner or present in your address book and is not a smart contract',
+            },
+            type: 'UNKNOWN_ADDRESS',
+          },
+          {
+            severity: 1,
+            address: recipient2,
+            description: {
+              short: 'Address is not known',
+              long: 'The address is not an owner or present in your address book and is not a smart contract',
+            },
+            type: 'UNKNOWN_ADDRESS',
+          },
+        ],
+      })
+    })
+
+    // Other
+    it('should warn about recipient of native transfer recipients', async () => {
+      const recipient = hexZeroPad('0x1', 20)
+
+      const safeTransaction = createSafeTransaction({
+        to: recipient,
+        data: '0x',
+      })
+
+      const result = await RecipientAddressModuleInstance.scanTransaction({
+        safeTransaction,
+        provider: mockProvider,
+        chainId: '1',
+        knownAddresses: [],
+      })
+
+      expect(isSmartContractSpy).toHaveBeenCalledTimes(1)
+      expect(mockGetBalance).toHaveBeenCalledTimes(1)
+      // Don't check as on mainnet
+      expect(mockGetSafeInfo).not.toHaveBeenCalled()
+
+      expect(result).toEqual({
+        severity: 1,
+        payload: [
+          {
+            severity: 1,
+            address: recipient,
+            description: {
+              short: 'Address is not known',
+              long: 'The address is not an owner or present in your address book and is not a smart contract',
+            },
+            type: 'UNKNOWN_ADDRESS',
+          },
+        ],
+      })
+    })
+  })
+
+  it('should not warn if the address(es) is/are used', async () => {
```

Comment:
What is the difference between this test and the previous one "should warn about recipient of native transfer recipients"?
They look identical to me.